controller.py

```diff
@@ -1,5 +1,6 @@
 from view import View
-from readchar import readkey, key
+from readchar import key
```

Comment:
Can't we get rid of `readchar` completely? Does `readkey` have it's own version of `key`?

Reply:
There's nothing in the documentation about checking keypresses so I don't think so? I've also tried importing key using readkeys and doesn't seem it's included.